.github/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 1. AIRAC (2407) - Changed color of Fast Aviation taxiway on Bournemouth (EGHH) SMR - thanks to @kristiankunc (Kristián Kunc)
 2. AIRAC (2407) - Updated Jersey (JSY) VOR coordinates - thanks to @chssn
 3. AIRAC (2407) - Removed Lashenden NDB (LSH) - thanks to @Bucklerz (Reece Buckley)
+4. AIRAC (2407) - Add new run up areas to the Redhill (EGKR) SMR - thanks to @kristiankunc (Kristián Kunc)
```

Comment:
```suggestion
4. AIRAC (2407) - Added new run up areas to the Redhill (EGKR) SMR - thanks to @kristiankunc (Kristián Kunc)
```
Changlog entry should be past tense :)

Reply:
Likewise, you marked this as resolved, but haven't made the change. I will commit the suggestion.